homeassistant/components/smappee/sensor.py

```diff
@@ -16,13 +16,6 @@
         "total_power",
         DEVICE_CLASS_POWER,
     ],
-    "total_reactive_power": [
```

Comment:
Is this a breaking change?

Reply:
This sensor is only available in rare cases (local, specific series). Reverted it for now, still need to look into (dis)abling the sensor.